libs/blocks/table/table.js

```diff
@@ -448,6 +457,15 @@ function applyStylesBasedOnScreenSize(table, originTable) {
     if ((!isMerch && !table.querySelector('.col-3'))
       || (isMerch && !table.querySelector('.col-2'))) return;
 
+    async function setAriaLabelsForElements(selectArray) {
+      const config = getConfig();
+      const ariaLabel = await replaceKey('choose-table-column', config);
+
+      selectArray.forEach((item) => {
+        item.setAttribute('aria-label', ariaLabel);
+      });
+    }
```

Comment:
This logic and the one from `setAriaLabelsForExpandableIcons`, are quite similar, I'd suggest we combine them into a more general method that we can call. Another tip: there is a `replaceKeyArray` with which you could get all the placeholders in one go, if that's helpful for the generic method.